src/components/container/ContainerComponent.tsx

```diff
@@ -88,6 +73,7 @@ export function ContainerComponent<TData = unknown>(
     getNewRoiData.current = props.getNewRoiData;
   }, [props.getNewRoiData]);
 
+  // @ts-expect-error types are wrongly exported by library
```

Comment:
I'm not sure about this error, but it seems like the library is not publishing the types correctly.

<img width="1748" height="600" alt="Image" src="https://github.com/user-attachments/assets/14726463-d37e-4cae-b747-1b9d5238818c" />

Reply:
Just use our helper. We should consolidate libraries (always use the same lib for the same usage)

Reply:
Can we resolve this and merge the PR?

I've created an issue about it.